modules/api-server/src/test/test.ts

```diff
@@ -14,9 +14,6 @@ const schoolsShortAPI = JSON.parse(fs.readFileSync(path.resolve("dist/test/expec
 const schoolsLongAPI = JSON.parse(fs.readFileSync(path.resolve("dist/test/expect/schools-long.json"), "utf-8"))
 const coursesShortAPI = JSON.parse(fs.readFileSync(path.resolve("dist/test/expect/courses-short.json"), "utf-8"))
 const coursesLongAPI = JSON.parse(fs.readFileSync(path.resolve("dist/test/expect/courses-long.json"), "utf-8"))
-const northwindAPI = JSON.parse(fs.readFileSync(path.resolve("dist/test/expect/northwind.json"), "utf-8"))

```

Comment:
Was it unused? Just like to understand

Reply:
Yes, they were unused, but I realised now, that the places where they are used are still in the code, but commented out. So maybe it would be better to keep them.